demo/helm/rook-operator/templates/deployment.yaml

```diff
@@ -0,0 +1,29 @@
+apiVersion: apps/v1beta1
+kind: Deployment
+metadata:
+  name: rook-operator
+  labels:
+    operator: rook
+    chart: "{{ .Chart.Name }}-{{ .Chart.Version | replace "+" "_" }}"
+spec:
+  replicas: 1
+  template:
+    metadata:
+      labels:
+        name: rook-operator
+        chart: "{{ .Chart.Name }}-{{ .Chart.Version | replace "+" "_" }}"
+    spec:
+      serviceAccountName: rook-operator
+      containers:
+      - name: rook-operator
+        image: "{{ .Values.image.repository }}:{{ .Values.image.tag }}"
+        imagePullPolicy: {{ .Values.image.pullPolicy }}
+        args: ["operator"]
+        env:
+        - name: ROOKD_REPO_PREFIX
+          value: quay.io/rook
```

Comment:
you'll want the .Values.image.repository here as well, but note that this prefix is missing the `rookd` repo name. Now I'm wondering if we should make these the same thing in the operator. The reason to have them be different is gone (we used to have rookd and rook-operator images, but now they are combined)